src/media.rs

```diff
@@ -0,0 +1,64 @@
+use std::{ffi::OsStr, path::Path};
+
+use async_graphql::{ComplexObject, Context, Error, Result, SimpleObject};
+use bson::Uuid;
+use s3::Bucket;
+use url::Url;
+
+/// Defines pre-signed URL expiration time of 1d.
+pub static PATH_EXPIRATION_TIME: u32 = 86400;
```

Comment:
imho this should be configurable via an env variable

Reply:
(of course you can use this value as fallback when no value is given)

Reply:
Will add that